lightning/src/routing/scoring.rs

```diff
@@ -388,19 +401,28 @@ pub struct ProbabilisticScoringParameters {
 	/// uncertainty bounds of the channel liquidity balance. Amounts above the upper bound will
 	/// result in a `u64::max_value` penalty, however.
 	///
+	/// `-log10(success_probability) * liquidity_penalty_multiplier_msat`
+	///
 	/// Default value: 30,000 msat
 	///
 	/// [`liquidity_offset_half_life`]: Self::liquidity_offset_half_life
 	pub liquidity_penalty_multiplier_msat: u64,
 
-	/// The time required to elapse before any knowledge learned about channel liquidity balances is
-	/// cut in half.
+	/// Whenever this amount of time elapses since the last update to a channel's liquidity bounds,
+	/// the distance from the bounds to "zero" is cut in half. In other words, the lower-bound on
+	/// the available liquidity is halved and the upper-bound moves half-way to the channel's total
+	/// capacity.
+	///
+	/// Because halving the liquidity bounds grows the uncertainty on the channel's liquidity,
+	/// penalties for payments that are within the liquidity bounds will be decreased. See the
```

Comment:
I think whether or not penalties are decrease depends on whether the amount lies closer to the upper bound or the lower bound.

Reply:
Would consider re-wording as:

"Because halving the liquidity bounds increases the uncertainty of the channel's liquidity, the penalty for an amount within the new bounds may increase or decrease depending on whether the new success probability decreased or increased, respectively."

Reply:
SGTM